roles/opensrp/tasks/main.yml

```diff
@@ -7,7 +7,7 @@
 - include_tasks: compile.yml
 
 - include_tasks: mysql.yml
-  when: opensrp_mysql_database is defined
+  when: opensrp_mysql_database is defined and opensrp_version is match('v1.\d.\d.\w+')
```

Comment:
@githengi what will happen `opensrp_version` is set to a branch name?

Reply:
This will not work for branches. I can configure a different variable for opensrp major version that can be used  for this check. Do have have suggestions for this

Reply:
why do we want to only call this task on a version deploy? is the task idempotent, is there a problem w/calling it more than necessary?

Reply:
This is because MYSQL tasks are not required in version of 2 of OpenSRP as all dependencies have been removed or migrated to PostgreSQL